examples/ruby-image-upload/README.md

```diff
@@ -0,0 +1,48 @@
+# Uploading Images to Cloudinary with the Ruby SDK
+
+## 🧰 Uploading with the SDK in a Ruby project
+
+To use the SDK in a Ruby project, first install the library:
+
+```
+gem install cloudinary
+```
+
+Then configure a new instance of Cloudinary:
+
+```
+development: 
+  cloud_name: "<Your Cloud Name>"
+  api_key: "<Your API Key>"
+  api_secret: "<Your Cloud Name>"
+  secure: true
+```
+
+Finally, run the upload method, passing in your image source:
+
+```
+image = './path/to/image.jpg' # Can be a local file path or a remote URL
+result = Cloudinary::Uploader.upload(image)
+```
+
+## 🚀 Get Started with This Example
+
+* Create an environment variable file `.env` and inside add:
+```
+CLOUDINARY_CLOUD_NAME="<Your Cloud Name>"
```

Comment:
You need to add the API KEY and API SECRET to the instructions, not just the cloud name, for getting the local example running

Reply:
Noted. I'll add the two as well